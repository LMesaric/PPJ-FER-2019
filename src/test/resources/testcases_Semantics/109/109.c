int main(void){

int a[1024] = "       f         h\s";
char b='p';
return 0;
}