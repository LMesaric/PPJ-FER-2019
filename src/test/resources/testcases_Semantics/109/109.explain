semantic: invalid escaped string '       f         h\s' on byte offset 19
note: '"       f         h\s"' on line 3
