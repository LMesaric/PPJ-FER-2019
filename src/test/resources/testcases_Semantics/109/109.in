<prijevodna_jedinica>
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 1 int
   IDN 1 main
   L_ZAGRADA 1 (
   KR_VOID 1 void
   D_ZAGRADA 1 )
   <slozena_naredba>
    L_VIT_ZAGRADA 1 {
    <lista_deklaracija>
     <lista_deklaracija>
      <deklaracija>
       <ime_tipa>
        <specifikator_tipa>
         KR_INT 3 int
       <lista_init_deklaratora>
        <init_deklarator>
         <izravni_deklarator>
          IDN 3 a
          L_UGL_ZAGRADA 3 [
          BROJ 3 1024
          D_UGL_ZAGRADA 3 ]
         OP_PRIDRUZI 3 =
         <inicijalizator>
          <izraz_pridruzivanja>
           <log_ili_izraz>
            <log_i_izraz>
             <bin_ili_izraz>
              <bin_xili_izraz>
               <bin_i_izraz>
                <jednakosni_izraz>
                 <odnosni_izraz>
                  <aditivni_izraz>
                   <multiplikativni_izraz>
                    <cast_izraz>
                     <unarni_izraz>
                      <postfiks_izraz>
                       <primarni_izraz>
                        NIZ_ZNAKOVA 3 "       f         h\s"
       TOCKAZAREZ 3 ;
     <deklaracija>
      <ime_tipa>
       <specifikator_tipa>
        KR_CHAR 4 char
      <lista_init_deklaratora>
       <init_deklarator>
        <izravni_deklarator>
         IDN 4 b
        OP_PRIDRUZI 4 =
        <inicijalizator>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       ZNAK 4 'p'
      TOCKAZAREZ 4 ;
    <lista_naredbi>
     <naredba>
      <naredba_skoka>
       KR_RETURN 5 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 5 0
       TOCKAZAREZ 5 ;
    D_VIT_ZAGRADA 6 }
