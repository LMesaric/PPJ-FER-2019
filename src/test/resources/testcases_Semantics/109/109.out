<primarni_izraz> ::= NIZ_ZNAKOVA(3,"       f         h\s")
